AU2/plugins/custom_plugins/CompetencyPlugin.py

```diff
@@ -299,23 +307,32 @@ def answer_show_inco_deadlines(self, _):
             competency_manager.add_event(e)
             death_manager.add_event(e)
 
-        deadlines = []
-        for a in ASSASSINS_DATABASE.get_identifiers():
-            deadlines.append((a, competency_manager.get_deadline_for(ASSASSINS_DATABASE.assassins[a])))
-            if "jackson" in a.lower():
-                competency_manager.is_inco_at(ASSASSINS_DATABASE.assassins[a], get_now_dt())
-        deadlines.sort(key=lambda t: t[1])
-
-        labels = []
-
         DATETIME_FORMAT = "%Y-%m-%d %H:%M"
 
-        labels.append(Label("__________________________________________________________________"))
-
-        for (a, d) in deadlines:
-            labels.append(Label(f"{a} -> {datetime.datetime.strftime(d, DATETIME_FORMAT)}"))
-            labels.append(Label("__________________________________________________________________"))
-        return labels
+        now = get_now_dt()
+        deadlines = []
+        for a in ASSASSINS_DATABASE.get_filtered(
+                include=lambda x: any(t.lower() in x.identifier.lower() for t in search_terms)
+        ):
+            d = competency_manager.get_deadline_for(a)
+            datetime_str = datetime.datetime.strftime(d, DATETIME_FORMAT) if d else "n/a"

```

Comment:
It would be useful if it gave some more info than just n/a. I've never had a good look at exactly how competency is managed in AU2, but the n/a s just don't tell me anything to aid debugging (as in, I wouldn't expect them ever, except on maybe police)

Reply:
That's a good point actually, it's odd that any assassins end up with n/a at all.